src/scripts/main.js

```diff
@@ -1,3 +1,261 @@
+  columnName.classList.add('active');
+});
+
+const formContainer = document.createElement('form');
+const nameLabel = document.createElement('label');
+const nameInput = document.createElement('input');
+const posiitionLabel = document.createElement('label');
+const posiitionInput = document.createElement('input');
+const officeLabel = document.createElement('label');
+const officeChoose = document.createElement('select');
+const ageLabel = document.createElement('label');
+const ageInput = document.createElement('input');
+const salaryLabel = document.createElement('label');
+const salaryInput = document.createElement('input');
+const formButton = document.createElement('button');
+const cityOptions = [
+  `Tokyo`,
+  `Singapore`,
+  `London`,
+  `New York`,
+  `Edinburgh`,
+  `San Francisco`,
+];
+const inputs = [nameInput, posiitionInput, officeChoose, ageInput, salaryInput];
+
+const modifyElement = (parentEl, child, cellName, type) => {
+  parentEl.innerHTML = `${cellName}:`;
+  parentEl.setAttribute('data-qa', cellName.toLowerCase());
+  child.required = true;
+  child.type = type;
+  parentEl.appendChild(child);
+};
+
+modifyElement(nameLabel, nameInput, 'Name', 'text');
+modifyElement(posiitionLabel, posiitionInput, 'Position', 'text');
+modifyElement(ageLabel, ageInput, 'Age', 'number');
+modifyElement(salaryLabel, salaryInput, 'Salary', 'number');
+
+formContainer.classList.add('new-employee-form');
+
+officeLabel.innerHTML = 'Office:';
+officeLabel.setAttribute('data-qa', 'office');
+officeLabel.appendChild(officeChoose);
+officeChoose.required = true;
+formButton.innerText = 'Save to table';
+
+cityOptions.forEach((el) => {
+  const option = document.createElement('option');
+
+  option.value = el;
+  option.innerText = el;
+  officeChoose.appendChild(option);
+});
+
+formContainer.appendChild(nameLabel);
+formContainer.appendChild(posiitionLabel);
+formContainer.appendChild(officeLabel);
+formContainer.appendChild(ageLabel);
+formContainer.appendChild(salaryLabel);
+formContainer.appendChild(formButton);
+
+pageBody.appendChild(formContainer);
+
+const pushNotification = (title, description, type) => {
+  const container = document.createElement('div');
+  const containerTitle = document.createElement('h2');
+  const containerDescr = document.createElement('p');
+
+  container.classList.add('notification');
+  container.setAttribute('data-qa', 'notification');
+  containerTitle.classList.add('title');
+
+  if (type === 'success' || type === 'error') {
+    container.classList.add('notification', type);
+  }
+  containerTitle.classList.add('title');
+  containerTitle.innerText = title;
+  containerDescr.innerText = description;
+
+  container.appendChild(containerTitle);
+  container.appendChild(containerDescr);
+  pageBody.appendChild(container);
+
+  setTimeout(() => {
+    container.style.display = 'none';
+  }, 3000);
+};
+
+formButton.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  if (
```

Comment:
The conditionals in the if statement can be simplified and made more readable by extracting them into a separate function or by using more descriptive variable names.